pages/design/composite-template-empty.md

```diff
@@ -1,37 +1,40 @@
 ---
+id: 
 name:
 rule_type: composite
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied | further testing needed
+    inapplicable: satisfied | further testing needed
 
-atomic_rules:
+input_rules:
   -
```

Comment:
Should there be a comment here, about how we refer to input rules? I would suggest by id